src/cy-reporter.js

```diff
@@ -36,6 +36,39 @@ var resultsFolder;
  * @returns {testcase record object}
  */
 
+function setConfiguration(options) {
+
+  console.debug('START: configuration & options:');
+
+  resultsFolder = 'results';
+  logsFolders = path.join('cypress', 'logs');
```

Comment:
Should not be hardcoded, but should be set similar to the other "output" (screenshots and video) folders

Also, the variable name should be `logsFolder` (singular)

Reply:
BTW, looking at https://github.com/archfz/cypress-terminal-report#logging-to-files there appears to be `config.projectRoot`. Shouldn't you be using that instead of figuring out `specRoot` out of `specPattern`?